mm/routines/product.py

```diff
@@ -15,3 +15,13 @@ def update_enum_product_attribute(self, sku, att_name, att_value):
                 sku, att_name, att_value)
 
         return record
+
+    def get_configurable_associated_products(self, store_id=None, is_active=None, is_visible=None):
+        rows = \
+            self.call(
+                'get_configurable_associated_products',
+                store_id,
+                int(is_active) if isinstance(is_active, bool) else None,
```

Comment:
if they pass garbage, I want the default to apply (None) while allowing None/True/False as valid values. 
How would you express it? Let me know if there is a more Pythonic way.